File: Java/Algorithm/src/com/softwarescares/Graphs/Graph/Graph.java
package com.softwarescares;

import java.util.*;

public class Graph
{

    public static ArrayList<ArrayList<Integer>> adjacency_list;

    // Create Graph

    public static ArrayList createGraph(int Vertices)
    {

         adjacency_list = new ArrayList<>();

        for (int i = 0; i < Vertices; i++)
        {
            adjacency_list.add(new ArrayList<>());
        }

        return adjacency_list;

    }

    // A utility function to add
    // edge in an undirected graph
    static void addEdge(int source, int vertex)
    {
        adjacency_list.get(source).add(vertex);
        //adjacency_list.get(vertex).add(source);
    }

    // A utility function to print the adjacency
    // list representation of graph

    static  void printGraph()
    {
        for (int i = 0; i < adjacency_list.size(); i++)
        {
            System.out.println("\nAdjacency list of vertex" + i);
            System.out.print("head");
            for (int j = 0; j < adjacency_list.get(i).size(); j++)
            {
              System.out.print(" -> " + adjacency_list.get(i).get(j));
            }
            System.out.println();
        }
    }

}

File: PHP/Data Structures/Graphs/Graph.php
<?php

class Graph
{
    public $adjacency_list = [];

    public function createGraph($vertices)
    {
        for ($i = 0; $i < $vertices; $i++)
        { 
            array_push($this->adjacency_list, []);
        }
    }

    public function addEdge($source, $vertex)
    {
        array_push($this->adjacency_list[$source], $vertex);
        array_push($this->adjacency_list[$vertex], $source);
    }

    public function printGraph()
    {
        print("\nAdjacent List Representation Of Graph");
        for ($i = 0; $i  < sizeof($this->adjacency_list); $i++)
        { 
            print("\nAdjacent nodes of vertex" . $i);
            for ($j = 0; $j < sizeof($this->adjacency_list[$i]); $j++)
            { 
                echo(" -> " . $this->adjacency_list[$i][$j]);
            }
        }
    }
}

//Graph::createGraph(5);
/*
$graph = new Graph();
$graph->createGraph(6);
$graph->addEdge(0, 1);
$graph->addEdge(0, 4);
$graph->addEdge(1, 2);
$graph->addEdge(1, 3);
$graph->addEdge(1, 4);
$graph->addEdge(2, 3);
$graph->addEdge(3, 4);

$graph->printGraph();
*/

?>